src/main/resources/database_dump.sql

```diff
@@ -155,6 +155,21 @@ CREATE TABLE `queue` (
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `member`
+--
+
+DROP TABLE IF EXISTS `member`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `member` (
+  `member_id` bigint(20) PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  `e-mail` varchar(255) NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `nickname` varchar(255) DEFAULT NULL,
+  `provider` varchar(10) NOT NULL,
+  `organization` varchar(255) DEFAULT NULL
+);
```

Comment:
Please add `ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=latin1;` here as in other tables.